package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@madie/madie-models",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
not sure if it makes any difference but running `npm install` updates the version package lock as well

Reply:
hmm...I forgot about that. I'm also not sure but I'll do that and open a PR to get that into Develop first

Reply:
package-lock is now updated